build/ci/op-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+OP_TEST_SCRIPT_URL=${OP_TEST_SCRIPT_URL-"https://raw.githubusercontent.com/operator-framework/operator-test-playbooks/master/upstream/test/test.sh"}
+
+bash <(curl -sL $OP_TEST_SCRIPT_URL) $*
+rc=$?
+echo "rc=$rc"
```

Comment:
consider moving this to the next line, when rc==0

Reply:
done